.gitignore

```diff
@@ -15,4 +15,6 @@ npm-debug.log
 node_modules
 
 dist
-build
\ No newline at end of file
+build
+
+.DS_Store
```

Comment:
@bil-io this should be part of your [global gitignore file](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore)